lib/formatter/error-json-formatter.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+module.exports = (errorResponse, genericFormatter) => {
+    // The default function here should not be called because this content
+    // type should be handled by genericFormatter.  So, if execution does
+    // come to this function then I don't know what `body` really is.
+    return genericFormatter.formatWithFallback((req, res, body, done) => {
+        errorResponse.createAsync("Unknown error", "4qsTYJa3").then((errorObject) => {
```

Comment:
Each error is given a random code in our codebase.  Every time that error is produced it will have the same garbage code as an identifier.  Developers using the API should be able to trust that these codes don't change unless the meaning of the error also changes.